service/src/main/kotlin/fi/espoo/evaka/shared/db/JdbiExtensions.kt

```diff
@@ -161,7 +159,7 @@ val dateRangeColumnMapper = PgObjectColumnMapper {
 }
 
 val timeRangeColumnMapper = PgObjectColumnMapper {
-    assert(it.type == "timerange")
+    assert(it.type == "timerange" || it.type == "timerange_non_nullable_range")
```

Comment:
This is a weird workaround to an issue where integration tests read array columns as `timerage_non_nullable_range` but running local env live mapped to `timerange`. Any ideas how to actually fix this?

Reply:
This looks ok to me and it's actually the original type that is wrong... `timerange` in the database allows nulls in both start and end, but this doesn't map correctly to our Kotlin `TimeRange` that doesn't allow nulls.